plugin-base/src/main/java/io/everytrade/server/parser/exchange/BlockchainApiTransactionBean.java

```diff
@@ -41,7 +41,7 @@ public BlockchainApiTransactionBean(Transaction transaction, String base, String
         feeAmount = Client.satoshisToBigDecimal(transaction.getFee()).abs();
     }
 
-    public TransactionCluster toTransactionCluster(boolean isWithFee) {
+    public TransactionCluster toTransactionCluster(boolean isBuyWithFee, boolean isSellWithFee) {
```

Comment:
Do not add method parameters to boolean toTransactionCluster().
Add additional parameters to constructor of BlockchainApiTransactionBean